README.md

```diff
@@ -1,6 +1,7 @@
 # Zulip API
 
-[![Build status](https://travis-ci.com/zulip/python-zulip-api.svg?branch=master)](https://travis-ci.com/github/zulip/python-zulip-api)
+[![Build status](https://github.com/zulip/python-zulip-api/workflows/build/badge.svg?branch=master)]
+(https://github.com/zulip/python-zulip-api/actions?query=workflow%3Abuild)
```

Comment:
Perhaps minor, but should this link be to master too?

Reply:
yes, you are right thanks for this.